skripchenkoveta/hometask-4/spec/my_hash_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require '../my_hash'
+require 'rspec'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe MyHash do
+  subject(:my_hash) { described_class.new }
+
+  before do
+    my_hash['li'] = 'za'
+    my_hash['my'] = 13
+    my_hash[:b] = 'name'
+    my_hash[404] = 'error'
+  end
+
+  describe '#[]=' do
+    context 'introduction of new elements key=>value' do
+      it 'return new elements in the hash' do
+        expect(my_hash[:c] = 10).to eq 10
+        expect(my_hash[1] = -3).to eq(-3)
+      end
+    end
+    it 'return key=>value' do
+      expect(my_hash['li']).to eq 'za'
+      expect(my_hash['my']).to eq 13
+    end
+    context 'when you initialize the hash incorrectly' do
+      it 'return error - ArgumentError' do
+        expect { my_hash(8) }.to raise_error(ArgumentError)
+        expect { my_hash[] = 'flower' }.to raise_error(ArgumentError)
+      end
+    end
+  end
+  describe '#[]' do
+    context 'when read by key' do
+      it 'return read by key=>value' do
+        expect(my_hash[:b]).to eq 'name'
+      end
+    end
+  end
+
+  describe '#delete' do
+    it 'remove element from the hash' do
```

Comment:
Below you test that `delete` method returns null. 
If you want to test that it removes element from the hash, show that after you call the `delete` method this key-value pair does not present anymore.